string start_stop
int8 cluster_number